### ISSUE TITLE
Add more CI workers

### ISSUE THREAD

**Issue by jimmykarily** [epinio/epinio#712]
We are splitting the test suite more (#716). For that to have any effect, we will need more GitHub Actions workers.

## Acceptance Criteria

- Improve how we run the workers (systemd server) so they come up after something bad happens
- Add at least 2 more workers (because we will have 1 more parallel task to run on each PR).

**Comment by jimmykarily** [comment]
Splitting happening here: #716

**Comment by svollath** [comment]
on compute3.paas.suse.de there could live like 12 VMs (8/16/32 VCPUs, 18 GB RAM).

Will do that in 3 phases ...
* add/redo 9 runners with 16 VCPUs, 24 GB RAM each (with setup-runner script and tmux) - DONE!
* document access and action-runner/tmux restart in confluence
* look into https://docs.github.com/en/actions/hosting-your-own-runners/configuring-the-self-hosted-runner-application-as-a-service

**Comment by svollath** [comment]
Updated confluence page for ~kwk/Self+hosted+github+runners with information on reconnect runners offline.
Of course, one of those runners - e.g. "tf6-gha-runner-2" could also be used to test with https://docs.github.com/en/actions/hosting-your-own-runners/configuring-the-self-hosted-runner-application-as-a-service

**Comment by jimmykarily** [comment]
I switched all runner to use the systemd service. I added this to the service template:

```
[Service]
Restart=always
RestartSec=5s
```

so the service will restart if something goes wrong or if someone accidentally kills it. Also I "enabled" it with `systemctl enable service_name_here` to make sure starts when the VM is re-started. I updated the confluence page with instructions on how to debug and re-create the setup.

I think this issue is done now.
